docs/source/jupyter/ontology_interface.ipynb

```diff
@@ -201,6 +201,28 @@
     "- When the keyword `reference_by_label` is set to `False` (disabled) or not set, the dot notation is a shorthand for fetching by suffix instead. This keyword is **disabled** in the `city` namespace."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"admonition important\">\n",
+    "\n",
+    "The dot notation supports IPython autocompletion. For example, when working on a Jupyter notebook, once the namespace has been imported, it is possible to get suggestions for the entity names by writing `namespace.` and pressing TAB. \n",
```

Comment:
For the green box, maybe put a title over it. For example:


**Tip**
The dot notation supports IPython autocompletion. For example, when working on a Jupyter notebook, once the namespace has been imported, it is possible to get suggestions for the entity names by writing namespace. and pressing TAB.

Reply:
I will try again to put a title. If you remember, on the SCRUM meeting we discussed that due to it being a notebook it was not straightforwad.

Reply:
This seems to work for me:
```
<div class="admonition important">
    <div>**Tip**</div>
    The full API specifictions of the import and export functions can be found in the [utilities API reference page](../api_ref.md).
</div>
```

Reply:
Got the right hack.

![grafik](https://user-images.githubusercontent.com/43052541/119841981-91195e80-bf06-11eb-94eb-791d4cb1abb6.png)

Reply:
Finally this is the best I could achieve.

![grafik](https://user-images.githubusercontent.com/43052541/119847685-7b5a6800-bf0b-11eb-9006-b59524b645bb.png)

I think it is not worth to invest more time in this hack. If you have any ideas to hide the \`\` from the LHS we can give them a try .